pyfolio/timeseries.py

```diff
@@ -31,6 +28,9 @@
 from .interesting_periods import PERIODS
 
 
+#####################################
```

Comment:
Yes, I actually started doing that already but backtracked because it wasn't clear that e.g. the rolling statistics shouldn't also go in there.